testsuite/pytests/sli2py_recording/test_multimeter_stepping.py

```diff
@@ -88,10 +89,14 @@ def build_net(model):
     """
 
     nest.ResetKernel()
-
+    if model.startswith("eprop_"):
+        nest.resolution = 1.0
+        nest.eprop_update_interval = 1.0
+        nest.eprop_learning_window = 1.0
```

Comment:
```suggestion
        nest.resolution = default_delay
        nest.eprop_update_interval = nest.resolution
        nest.eprop_learning_window = nest.resolution
```

Reply:
Does one need to set the last two at all here?

Reply:
No, they can be deleted.